### ISSUE TITLE
Visual Timeline doesn't work

### ISSUE THREAD

**Issue by CarlosMoralesSal** [cyb3rfox/Aurora-Incident-Response#82]
I am testing the tool with two dump rows to visualize the timeline and it doesn't work. When I click on the Visual Timeline, it appears on grey and it doesn't appear anything. On timeline I have this
![CapturaTL](https://user-images.githubusercontent.com/13416037/120883725-1673d180-c5df-11eb-90a2-bdc02b4456b3.PNG).

What happen?, why the timeline doesn't appear?. Thanks. I am working in a Ubuntu 20 and trying to work with develop mode

**Comment by cyb3rfox** [comment]
Can you share  the datafile and which version you use?

**Comment by CarlosMoralesSal** [comment]
Yes, I share the .fox and the csv. Aurora's version, it is the latest as I did a git clone yesterday. Thanks
[aurora.zip](https://github.com/cyb3rfox/Aurora-Incident-Response/files/6602816/aurora.zip)

**Comment by cyb3rfox** [comment]
Thanks now I can see it. Dates need to be in ISO format. YYYY-MM-DD hh:mm:ss. Cheers

**Comment by CarlosMoralesSal** [comment]
Hi @cyb3rfox , even if I have modified settings parameters, should be always in ISO format?

**Comment by cyb3rfox** [comment]
The parameters in settings.js only affect the UI components at this point. The logic that draws the timeline expects it to be ISO. I'll open that again as a feature request. So that settings.js affects the timeline as well and converts it. However ISO is standard for most investigators as it also sorts correctly as text.
